test/c_api/flat_tuple_test.cpp

```diff
@@ -54,8 +54,8 @@ TEST_F(CApiFlatTupleTest, ToString) {
     columnWidths[0] = 10;
     columnWidths[1] = 5;
     columnWidths[2] = 10;
-    auto str = kuzu_flat_tuple_to_string(flatTuple, columnWidths, 3, "\t", 1024);
-    ASSERT_EQ(std::string(str), " Alice    \t 35  \t 1.731000 ");
+    auto str = kuzu_flat_tuple_to_string(flatTuple);
```

Comment:
Since the function no longer requires passing a `columnWidths` array. It should also be removed from this test case.

Reply:
This test is meant to replace the original to_string API with a new one that doesn't require columnWidths. to_string() is still needed.

Reply:
I mean that the lines:
```
    auto columnWidths = (uint32_t*)malloc(3 * sizeof(uint32_t));
    columnWidths[0] = 10;
    columnWidths[1] = 5;
    columnWidths[2] = 10;
```
and
```
    free(columnWidths);
```
should be removed as the function is no longer called with `columnWidths`.